.github/workflows/cifuzz.yml

```diff
@@ -0,0 +1,35 @@
+name: CIFuzz
+on: [pull_request]
```

Comment:
I guess this only needs to run when stuff under `./src` changes, even then we may be able to limit it some more as we currently only cover some specific areas...

So at the very least

```yaml

  paths:
    - 'src/**'
```

Reply:
Will this [patch](https://github.com/nginx/unit/pull/1327/files#diff-cb0618c14c125bb79d60ba9a127ed18f4a3ce62cff9c49e26c8890afb6f35668R2-R6) work?

I have another follow-up patch on my disk. I will apply it as soon as this one looks good to you.

Reply:
I wonder if it should also include the fuzzing stuff itself... `fuzzing/**`